src/graphql/resolvers.ts

```diff
@@ -357,6 +373,18 @@ const Query: QueryResolvers = {
 
     return results[0];
   },
+  cubeHealth: async () => {
+    const cube = await getObservationsCube();
+    const dimensions = cube.dimensions.map((d) => d.path.value);
+    const missingDimensions = new Set(expectedCubeDimensions);
+    for (const dim of dimensions) {
+      missingDimensions.delete(dim);
+    }
```

Comment:
Maybe you could use d3.difference (https://github.com/d3/d3-array#difference) here? I see that this project uses d3 :)

Reply:
Good idea, thanks, done